frontend/openchat-client/src/openchat.ts

```diff
@@ -4247,9 +4248,9 @@ export class OpenChat extends OpenChatAgentWorker {
         }
     }
 
-    clearSelectedCommunity(): void {
-        selectedCommunityId.set(undefined);
-    }
+    // clearSelectedCommunity(): void {
```

Comment:
Is this meant to be included?

Reply:
Yes - I don't _think_ we need it and it certainly won't cause any immediate problem.